javaslang-benchmark/src/test/java/javaslang/JmhRunner.java

```diff
@@ -1,44 +1,44 @@
-package javaslang.benchmark;
+package javaslang;
 
+import javaslang.collection.Array;
 import org.openjdk.jmh.annotations.Mode;
 import org.openjdk.jmh.results.RunResult;
 import org.openjdk.jmh.runner.Runner;
 import org.openjdk.jmh.runner.options.*;
 
-import java.util.*;
+import java.util.Random;
 import java.util.concurrent.TimeUnit;
+import java.util.function.*;
 
 public class JmhRunner {
-    private static final int FORKS = 1;
+    public static boolean WITH_ASSERTS = false;
```

Comment:
are we sure that there runs only one JmhRunner per JVM? Otherwise we need instances of JmhRunner.